src/js/providers/html5.ts

```diff
@@ -1012,15 +1028,17 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
         return false;
     }
 
-    function clearTimeouts() {
+    function clearTimeouts(): void {
         clearTimeout(_staleStreamTimeout);
         _staleStreamTimeout = -1;
     }
+
+    return this;
```

Comment:
We don't need to return `this`. The return type for function invoked as a constructor with the `new` keyword is `void` .